docs/reference-docs/workflows/workflows.md

```diff
@@ -0,0 +1,18 @@
+Workflows are what actually takes a product definition and populates your domain models.
```

Comment:
Missing a header?

Reply:
I guess it uses like the filename if there isn't a main header, still beter to add one